.github/CONTRIBUTING.md

```diff
@@ -47,6 +47,14 @@ git push -u origin feat-my-awesome-new-feature
 git remote add upstream git@github.com:adobe/spectrum-css.git
 ```
 
+To ensure your Node environment is always aligned with the project, we strongly recommend using [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating). Once you have nvm installed, you can run:
```

Comment:
The changes in this doc were moved over from where they used to live in the README.md to provide a more thorough explanation in a central place